### DISCUSSION TITLE
Welcome to AI-on-the-edge-device Discussions!

### DISCUSSION THREAD

**Discussion by jomjol** [jomjol/AI-on-the-edge-device#599] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Jonvelle53** [comment]
Hallo - bin im Ruhestand und hab nun endlich die 70 erreicht ;(
Vertreibe mir nun die Zeit mit Home Assistant, 0-Ahnung von den hier benötigten Programmier Kenntnissen, aber ich arbeite mich ein.
Bin begeistert von all den Möglichkeiten, die Ihr hier vorstellt mit viel Liebe und Geduld den Ahnungslosen hier nahe bringt.

Insbesondere in meinem Fall im Moment vielen Dank für den Wasserzähler.

Habe alles besonders auf Grund der perfekten Dokumentation und des automatisierten Ablaufs über das Webinterface nachbauen und im Home Assistant ausgeben können.

Eine Frage habe ich allerdings dennoch - >

Wie bekomme ich den falschen Vorwert geändert? Auf der Webseite kann ich bei Update unten nichts eingeben. Der falsche wert ist mir bei der Einrichtung der Digits entstanden.
Eventuell muss ich ja auch nur ein wenig warten, und das Ganze erledigt sich wie von Zauberhand von alleine?

Dann bis Denne - wie der Berliner zu sagen pflegt.
Jonvelle

**Comment by Jonvelle53** [comment]
Also - man sollte etwas genauer hinsehen und das englische dann auch verstehen ;-)
Habe es natürlich einfach über den Button "Update value" gelöst.
Ich hatte das vorher falsch verstanden.
Viele Grüße
Jonvelle

**Comment by markusruebe** [comment]
Hello everyone, I am currently getting the IP assigned to my AI in the Edge ESP CAM via DHCP. How can I subsequently enter the IP as a fixed IP? I can't find anything about this in the config. Thanks for a hint ;-)

**Comment by Jonvelle53** [comment]
Einfach im Router festlegen (z.B. Fritz.Box(

 

Von: Markus R. ***@***.***> 
Gesendet: Sonntag, 17. November 2024 13:12
An: jomjol/AI-on-the-edge-device ***@***.***>
Cc: Jonvelle53 ***@***.***>; Comment ***@***.***>
Betreff: Re: [jomjol/AI-on-the-edge-device] Welcome to AI-on-the-edge-device Discussions! (Discussion #599)

 

Hallo zusammen, ich bekomme aktuell für meine AI in the Edge ESP CAM die IP per DHCP zugewiesen. Wie kann ich die IP nachträglich als feste IP eintragen? Ich finde hierzu nichts in der Config. Danke für einen Hinweis ;-)

**Reply by markusruebe** [reply]
ok - das bekomme ich hin ;-) ich dachte man könnte das ggf. über das Web Frontend auf dem ESP einstellen. Danke für die schnelle Antwort!

**Reply by SybexX** [reply]
Man kann eine feste IP in die wlan.ini eintragen, ich würde aber empfehlen, es in der Fritzbox zu machen.
Über das Web Frontend geht das nicht, man muss die wlan.ini auf der SD am PC bearbeiten.

![Unbenannt](https://github.com/user-attachments/assets/454c904a-45f5-47ef-98c5-11b7084adc91)

**Reply by markusruebe** [reply]
danke!

**Reply by markusruebe** [reply]
Ich habe das jetzt über den DHCP server geregelt, dann muss ich die Sd Karte nicht rausfummeln ;-)

**Comment by sfadriaan** [comment]
Hi everyone! 👋🏽 

Excited to implement this tool! I will use it to read my electricity utility meter to ultimately track my usage via Home Assistant.

Thanks for all the effort and sharing this with the world! 👍🏽
